bin/benchpark

```diff
@@ -86,10 +116,14 @@ def benchpark_setup_handler(args):
     benchmark = args.benchmark
     system = args.system
     workspace_root = pathlib.Path(os.path.abspath(args.workspace_root))
-
     source_dir = source_location()
-
     debug_print(source_dir)
+    debug_print(benchmark)
+    valid_benchmark = benchpark_check_benchmark(benchmark)
+    debug_print(system)
+    valid_system = benchpark_check_system(system)
+    if not (valid_benchmark and valid_system):
+        return
```

Comment:
I recommend having these functions just raise an exception like `BenchmarkSpecificationException`, and we can catch here if we want to print out something cleaner than a stack trace (this strategy works well within Spack at least, and can accommodate less-structured problems that should result in a termination).

Reply:
I think you have something specific in mind; rather than me trying to guess, do you mind handling this?